src/ikoner/SlettetIkon.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+interface Props {
+    className?: string;
+    heigth?: number;
+    width?: number;
+}
+
+const SlettetIkon: React.FC<Props> = ({ className, heigth, width }) => {
+    return (
+        <svg
+            width={width}
+            height={heigth}
+            className={className}
+            viewBox="0 0 24 24"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            focusable="false"
+            role="img"
+        >
+            <path
+                fill-rule="evenodd"
+                clip-rule="evenodd"
```

Comment:
Får feilmelding om dette ved kall mot htmlify
```suggestion
                fillRule="evenodd"
                clipRule="evenodd"
```